build.yaml

```diff
@@ -3187,8 +3194,8 @@ targets:
   deps:
```

Comment:
Btw, the order of dependencies in the original code 
https://github.com/grpc/grpc/blob/6b6954050cf0379dbbee90dd18313a3779e0dd52/build.yaml#L3204 (it should be mostly the reverse, as for all the other targets).